Pod/Native/NativeFeature.swift

```diff
@@ -9,16 +9,55 @@
 import Foundation
 
 private struct FileTags {
-    static let Feature = "Feature:"
-    static let Background = "Background:"
-    static let Scenario = "Scenario:"
-    static let Outline = "Scenario Outline:"
-    static let Examples = "Examples:"
-    static let ExampleLine = "|"
-    static let Given = "Given"
-    static let When = "When"
-    static let Then = "Then"
-    static let And = "And"
+    static var Feature: [String] { return localized(expression: "feature", default: "Feature:") }
+    static var Background: [String] { return localized(expression: "background", default: "Background:") }
+    static var Scenario: [String] { return localized(expression: "scenario", default: "Scenario:") }
+    static var ScenarioOutline: [String] { return localized(expression: "scenarioOutline", default: "Scenario Outline:") }
+    static var Examples: [String] { return localized(expression: "examples", default: "Examples:") }
+    static let ExampleLine: [String] = ["|"]
+    static var Given: [String] { return localized(expression: "given", default: "Given ") }
+    static var When: [String] { return localized(expression: "when", default: "When ") }
```

Comment:
Why do the default values have a trailing space - I'd rather that wasn't a requirement of the translated values if possible.

Reply:
They should because there are some languages where words have the same prefix, i.e. `To` and `Тогда` for Russian `Then`.  Without space when longer keyword is used we will parse its remaining part as expression. Space ensures that the whole word is matched. And also this json file I used is exactly what cucumber uses so I didn't change anything, except removing some unneeded keys to simplify parsing.